src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "products")
+@Setter
+@Getter
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private Category category;
+
+    @Override
```

Comment:
you can use @ToString

Reply:
Yes, but I relied on the materials described in our lecture:
![image](https://github.com/mate-academy/jv-springboot-data-jpa/assets/112614843/de242881-24ff-4994-8f86-5ab6ffc476de)